src/components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+const Footer = () => {
+  return (
+    <footer className="about">
```

Comment:
In the future, we may want to consider throwing this into the layout component so it can be used across all pages.